CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.5.0 / 2016-11-01
+
+This release requires a storage wipe. It contains fundamental internal
+changes that came with implementing the high availability mode.
+
+* [FEATURE] Alertmanager clustering for high availability
+* [FEATURE] Garbage collection of old silences and notification logs
+* [CHANGE] New storage format
+* [CHANGE] Stricter silence semantics for consistent historical view
+
 ## v0.5.0-beta.0 / 2016-10-06
```

Comment:
Not feeling strongly, just trying to document the practice so far. If you like to change it, go ahead. It should just be an informed decision.